tests/llmobs/_utils.py

```diff
@@ -268,7 +268,7 @@ def _completion_event():
         "tags": ["version:", "env:", "service:tests.llmobs", "source:integration"],
         "start_ns": 1707763310981223236,
         "duration": 12345678900,
-        "error": 0,
```

Comment:
🤔 why did this change?

Reply:
This was an outdated fixture ([context](https://github.com/DataDog/dd-trace-py/pull/12644#discussion_r1985665120): we switched from `error` to `status` as the primary span event field)